ImageManipulator/ImageManipulator/ImageManipulator/Filters.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  Filters.swift
+//  ImageManipulator
+//
+//  Created by Lambda_School_Loaner_268 on 5/4/20.
+//  Copyright © 2020 Lambda. All rights reserved.
+//
+
+import Foundation
+import UIKit
+import CoreImage
+import CoreImage.CIFilterBuiltins
+import Photos
+
```

Comment:
I really like this interface! You might even consider using an enum that has a string raw value, which would make it even more type-safe (eg, an enum `FilterType` that has a case for each of the string cases here, and then uses the rawValue to actually initialize the `CIFilter`).